swap_meet/decor.py

```diff
@@ -1,2 +1,24 @@
-class Decor:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Decor(Item):
+    def __init__(self, id=None, width=0, length=0, condition=0):
+        super().__init__(id, condition)
+        self.width = width
+        self.length = length
+        self.condition = condition
+
+    def get_category(self):
+        # return self.__class__.__name__
+        return "Decor"
+    
+    def __str__(self):
+        id = self.id
+        width = self.width
+        length = self.length
+        result = f"An object of type Decor with id {id}. It takes up a {width} by {length} sized space."
+        return result
```

Comment:
Remove unneeded variables and directly access instance attributes to keep your code concise.

```suggestion
    def __str__(self):
        result = f"An object of type Decor with id {self.id}. It takes up a {self.width} by {self.length} sized space."
        return result
```

Reply:
Like my comment in `Clothing`, how could you refactor `__str__` in the parent class and leverage the functionality in the parent class's method so you can avoid repeated logic here?